src/api/contracts.py

```diff
@@ -0,0 +1,92 @@
+from sanic import Blueprint, response
+
+from src.models.contract import Contract
+from src.network import chains
+from src.process import processor
+from src.process.utils import restart_collectors
+
+contracts_bp = Blueprint("contracts_blueprint", url_prefix="/api/contracts")
+
+
+@contracts_bp.route("/")
+async def get_all_contracts(request) -> response:
+    """Get all contracts.
+
+    Args:
+        request: request
+
+    Returns:
+          response: contracts payload
+    """
+    contracts = await Contract.get_contracts(request.app.db)
+    return response.json(
+        {"contracts": [contract.serialise() for contract in contracts]}
+    )
+
+
+@contracts_bp.route("/<chain>")
+async def get_chain_contracts(request, chain: str) -> response:
+    """Get contracts for specified chain.
+
+    Args:
+        request: request
+        chain: chain name
+
+    Returns:
+        response: contracts payload
+    """
+    contracts = await Contract.get_contracts(request.app.db, chain)
+    return response.json(
+        {"contracts": [contract.serialise() for contract in contracts]}
+    )
+
+
+@contracts_bp.post("/")
+async def create_contract(request) -> response:
+    """Create contract.
+
+    Args:
+        request: request
+
+    Returns:
+        response: response
+    """
+    data = request.json
+    await Contract.create_contract(
+        request.app.db, data["id"], data["active"], data["chain"]
```

Comment:
Data should be validated:
- is the given address of a correct format depending on the chain?